py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -24,6 +24,36 @@
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 
+# nearest, linear, cubic
+class GridSamplerInterpolation:
+    def __init__(self):
+        self.interpolator_mode = None
+
+    def __call__(self, interpolator_int):
+        if interpolator_int == 0:
+            self.interpolator_mode = trt.InterpolationMode.NEAREST
+        elif interpolator_int == 1:
+            self.interpolator_mode = trt.InterpolationMode.LINEAR
+        elif interpolator_int == 2:
+            self.interpolator_mode = trt.InterpolationMode.CUBIC
+        return self.interpolator_mode
+
+
+# zeros, border, reflection
+class GridSamplerSampling:
+    def __init__(self):
+        self.sample_mode = None
+
+    def __call__(self, sample_int):
+        if sample_int == 0:
+            self.sample_mode = trt.SampleMode.FILL
+        elif sample_int == 1:
+            self.sample_mode = trt.SampleMode.CLAMP
+        elif sample_int == 2:
+            self.sample_mode = trt.SampleMode.REFLECT
+        return self.sample_mode
```

Comment:
I think this could just go in the `grid.py` file, since it is very specific to the grid operations.